### PR TITLE
Garklavs cos post

### PR DESCRIPTION
**Pull request by kategarklavs**
This is Ty Grandison's CareerOneStop post (to be published EOD Monday).

It still needs an image!

### CODE REVIEW

File: _posts/2015-03-02-cos-post.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: New Resources at CareerOneStop
+layout: post
+image:
```
Comment (gboone) [inline _posts/2015-03-02-cos-post.md]: @kategarklavs you can either remove this line or change it to `image: ""` until we get an image to put here.

File: _posts/2015-03-02-cos-post.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: New Resources at CareerOneStop
+layout: post
+image: 
+tags:
+- PIF
+- CareerOneStop
+author:
+- ty
```
Comment (gboone) [inline _posts/2015-03-02-cos-post.md]: this should be `tgrandison`

File: _posts/2015-03-02-cos-post.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+excerpt: "When the Employment and Training Administration’s CareerOneStop team set out to redesign the career, training, and job resources sections of the CareerOneStop site, they didn’t immediately begin rewriting code. Instead, they embraced a user-centered approach that focused on the user experience (UX). Read on to learn more about their findings."
+description: "When the Employment and Training Administration’s CareerOneStop team set out to redesign the career, training, and job resources sections of the CareerOneStop site, they didn’t immediately begin rewriting code. Instead, they embraced a user-centered approach that focused on the user experience (UX). Read on to learn more about their findings."
+---
+
+<p class="authors">
+  by {% author ty %}
```
Comment (gboone) [inline _posts/2015-03-02-cos-post.md]: `ty` should be `tgrandison`

File: _posts/2015-03-02-cos-post.md (hunk 2 above)
Comment (kategarklavs) [inline _posts/2015-03-02-cos-post.md]: Fixed!

File: _posts/2015-03-02-cos-post.md (hunk 1 above)
Comment (kategarklavs) [inline _posts/2015-03-02-cos-post.md]: Done — I added the image (and fingers crossed that it appears correctly)

**Comment by gboone** [comment]
It has an image!

**Comment by kategarklavs** [comment]
Huzzah(!)

**Pull request merged**